tox.ini

```diff
@@ -6,7 +6,7 @@ envlist = py36
 passenv = LC_ALL, LANG, HOME
 deps =
     binaryornot
-    flake8==2.5.5
+    flake8==3.5.0
```

Comment:
I suggest that we address that with a straightforward

```ini
deps = -rrequirements.txt
```

therefore closing #1494.